app/helpers/ems_middleware_helper/textual_summary.rb

```diff
@@ -11,7 +11,7 @@ def textual_group_properties
   def textual_group_relationships
     # Order of items should be from parent to child
     items = []
-    items.concat(%i(middleware_servers middleware_deployments middleware_datasources))
+    items.concat(%i(middleware_servers middleware_deployments middleware_datasources middleware_messagings))
```

Comment:
I think you are correct. `%i(...)` already creates an array of symbols, should be sufficient. 
(and yes, it was not added in this PR, so it can be certainly fixed in a separate followup)